pyproject.toml

```diff
@@ -4,30 +4,44 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "cupyd"
-version = '0.1.0'
+dynamic = ["version"]

```

Comment:
this way, we don't need to bump the version but will be automatically taken from the tag release